src/recipe/test/e2e/createRecipe.controller.e2e-spec.ts

```diff
@@ -74,6 +74,7 @@ describe('Recipe Controller - Create', () => {
     prismaService = moduleFixture.get<PrismaService>(PrismaService);
 
     testUser = await userService.generateAccount(
+      process.env.TEST_EMAIL,
```

Comment:
what do you mean exactly?

Reply:
you keep calling `userService.generateAccount` in multiple places with the same arguments to create a test account.
Now you have changed the generateAccount method signature so you have to update it everywhere. 
 
This is something that you want to avoid and looks like something that can be abstracted into different method. eg: `createTestAccount()`. Not using the `UserService` for that would also be a good idea. You could use Prisma client for that.

Also, why do you still need to create a test user at startup in main.ts? Keep in mind that those users will be created each time you redeploy the app and the email must be unique.

I'm not saying you need to change it right now. I only want to bring it to your attention.